configs/siklu_80X0_defconfig

```diff
@@ -4,7 +4,7 @@ BR2_ARM_FPU_VFPV4=y
 BR2_SVN="svn"
 BR2_DL_DIR="${CACHE_ROOT_DIR}/cache/siklu/portf/80XX/buildroot-dl"
 BR2_PRIMARY_SITE="${BUILDROOT_80XX_MIRROR}"
-BR2_PRIMARY_SITE_ONLY=y
+#BR2_PRIMARY_SITE_ONLY is not set
```

Comment:
We need to allow download from the nexus server only (to prevent jenkins accessing internet)
We need to add sftp server we are going to use to the nexus repo.
To do that, please download the source package we need to copy to the nexus server from the original location and send it to me

Reply:
@alanyanigersiklu 
What Dima is saying is that we need to reverse the order of operations - first add the Buildroot sftp package to the nexus mirror, then create the topic branch.